src/colorTypes.ts

```diff
@@ -0,0 +1,55 @@
+type ForegroundColors =
+  | "black"
+  | "blackBright"
+  | "blue"
+  | "blueBright"
+  | "cyan"
+  | "cyanBright"
+  | "gray"
+  | "green"
+  | "greenBright"
+  | "grey"
+  | "magenta"
+  | "magentaBright"
+  | "red"
+  | "redBright"
+  | "white"
+  | "whiteBright"
+  | "yellow"
+  | "yellowBright";
+
+type BackgroundColors =
+  | "bgBlack"
+  | "bgBlackBright"
+  | "bgBlue"
+  | "bgBlueBright"
+  | "bgCyan"
+  | "bgCyanBright"
+  | "bgGray"
+  | "bgGreen"
+  | "bgGreenBright"
+  | "bgGrey"
+  | "bgMagenta"
+  | "bgMagentaBright"
+  | "bgRed"
+  | "bgRedBright"
+  | "bgWhite"
+  | "bgWhiteBright"
+  | "bgYellow"
+  | "bgYellowBright";
+
+type Modifiers =
+  | "blink"
+  | "bold"
+  | "dim"
+  | "doubleunderline"
+  | "framed"
+  | "hidden"
+  | "inverse"
+  | "italic"
+  | "overlined"
+  | "reset"
+  | "strikethrough"
+  | "underline";
+
+export type Color = ForegroundColors | BackgroundColors | Modifiers | Array<ForegroundColors | BackgroundColors | Modifiers>;
```

Comment:
```suggestion
export type Color = ForegroundColors | BackgroundColors | Modifiers;
```
I think it's better to use `Color | Color[]` in the code than forcing the type to natively accept array